kubeflow/common/cert-manager/Makefile

```diff
@@ -0,0 +1,12 @@
+
```

Comment:
add a default build_dir, so that the Makefile can run on its own too?

```
build_dir?=./build
```

Reply:
Make sense, done.